roles/apache/templates/serverspec/apache_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+describe package('apache2') do
+  it { should be_installed }
+end
+
+describe package('lynx-cur') do
+  it { should be_installed }
+end
+
+['000-default', '000-default.conf', 'default', 'default.conf'].each do |file|
+  describe file("/etc/apache/sites-enabled/#{file}") do
+    it { should_not exist }
+  end
+end
+
+describe file('/etc/apache2/ports.conf') do
+  it { should be_file }
```

Comment:
You have a task explicitly removing this file, so I'm confused by the server spec for it.